docs/src/main/asciidoc/index.adoc

```diff
@@ -36,6 +35,11 @@ you how to create a cluster, how configure the web container and EJB
 container for clustering, and shows you how to configure load balancing
 and failover
 
+== Model Reference
+
+* The link:/wildscribe[Wildscribe model reference^] provides information about all subsystem
```

Comment:
When I build and open target/generated_docs/index.html, this link is broken.

Reply:
Oops yeah. I had it fixed locally and forgot to push it up.